pages/studentPortal/challenges/dungeonsAndDragonsCharacterSheet/LuckySubmission/levelOne.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { ATTRIBUTE_LIST } from "../constants";
+const levelOne = () => {
+  // PSEUDOCODE
+  // create a row with 6 attributes
+  // create a list of the 6 attribute names
+  // create a useState hook to track the attributes count independently
+  // map each attribute, which should have a counter at 0 and the attribute name
+  // each attribute has two buttons below to inc and dec that count
+
+  // CONSTANTS
+  const attributes = ATTRIBUTE_LIST;
+
+  // STATE VARIABLES
+  const [attributeCount, setAttributeCount] = useState([0, 0, 0, 0, 0, 0]);
```

Comment:
I might rename this variable to be more clear that this is an array of counts. Not just one count. 

Maybe attributeCountArray or attributeCounts

Reply:
ah right, my bad. changed to `attributedCounts `and `setAttributeCounts`